dig.go

```diff
@@ -125,6 +139,9 @@ type Container struct {
 
 	// Source of randomness.
 	rand *rand.Rand
+
+	// Dot graph of dependencies
+	dotgraph []graphEdge
```

Comment:
nit: `dotgraph` -> `dotGraph`. We might also want to consider making:
```
type dotGraph struct { ... }
```
for the `[]graphEdge` representation.

This could become (or something similar):

```
type Container struct {
  ...
  dg *dotGraph
}
```